### PR TITLE
Jenkins playbooks

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
This moves https://github.com/chrismeyersfsu/playbook-ansible_integration into the ansible repo.

The playbook is already running on Jenkins as `Test_Ansible_Integration_Easy_And_Simple`. This PR is also running on Jenkins as `Test_Ansible_Integration_Easy_And_Simple_chrismeyersfsu_ansible_jenkins-playbooks`. This shows that once this PR is accepted, a we need only flip the repo configured in Jenkins.

### CODE REVIEW

**Comment by bcoca** [comment]
just to clarify, we should rename the role to ansible_test_deps

**Comment by chrismeyersfsu** [comment]
@bcoca renamed.

File: test/utils/ansible-playbook_integration_runner/inventory [hunk 1]
```diff
@@ -0,0 +1 @@
+localhost ansible_connection=local ansible_python_interpreter="/usr/bin/env python"
```
Comment (bcoca) [inline test/utils/ansible-playbook_integration_runner/inventory]: this breaks on several platforms, leave it up to user to set at top level

File: test/utils/ansible-playbook_integration_runner/inventory.dynamic [hunk 2]
```diff
@@ -0,0 +1,3 @@
+localhost ansible_connection=local ansible_python_interpreter="/usr/bin/env python"
```
Comment (bcoca) [inline test/utils/ansible-playbook_integration_runner/inventory.dynamic]: same as above, don't specify ansible_python_interpreter here

**Comment by chrismeyersfsu** [comment]
@bcoca Anything else?
Note: I am syncing the subtree to https://github.com/chrismeyersfsu/ansible_test_deps to invoke automated travisci tests for the ansible_test_deps role. The results are visible from the README badge https://travis-ci.org/chrismeyersfsu/ansible_test_deps

**Pull request merged**